lenny/schemas/waitlist.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class WaitlistBase(BaseModel):
+    book_id: int
+    user_id: int
+
+
+class WaitlistCreate(WaitlistBase):
+    pass
```

Comment:
It's not a race to publish the most code, let's open smaller PRs with more complete logic, otherwise it belongs in a draft or on another branch. :)